imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -63,6 +77,15 @@ class DfmMetaMod(Driver):
     # masks used for  modflow-dflow exchanges
     mask_active_mod_dflow1d = dict[str, NDArray[np.int_]]()
 
+    # dictionary with mapping tables for mod=>msw coupling
+    map_mod2msw: Dict[str, csr_matrix] = {}
```

Comment:
Initializing with a dict at that level turned out to be problematic.
Please change that similar to what I did in the last commit of lumbricus: https://github.com/Deltares/imod_coupler/commit/a798c9c8071cf7f0fe03c96bce48013b1a906abe

Reply:
ok, its changed. For my information, why did this give problems? I can see at least it gives cleaner code.

Reply:
The problem is that the former code resulted in class variable set to a single dictionary.
Imagine the following situation with two different instances of `DfmMetaMod`:
```python
>>> driver_1 = DfmMetaMod()
>>> driver_2 = DfmMetaMod()
>>> driver_1.map_mod2msw["blah"] = "whooo"
>>> print(driver_2.map_mod2msw["blah"])
whooo
```

Not a situation that will happen very often, but still a bug.
And it is only a problem if you set the default value of a class variable to a mutable container like dict or a numpy array.